src/pages/HomePage/components/SearchPanel/SearchPanel.tsx

```diff
@@ -0,0 +1,74 @@
+import { Component, createRef } from 'react';
+import './SearchPanel.css';
+import bookImg from './book-gif.svg';
+import magnifyingGlassIcon from './magnifying-glass-svgrepo-com.svg';
+import binocularsIcon from './Binoculars.svg';
+import libraryImg from './BannerImg.svg';
+
+interface SearchPanelProps {
+  onSearch: (searchTerm: string) => void;
+  searchTerm?: string;
+}
+class SearchPanel extends Component<SearchPanelProps> {
+  private inputRef = createRef<HTMLInputElement>();
+  constructor(props: SearchPanelProps) {
```

Comment:
I prefer to add an empty line between them

Reply:
done